README.md

```diff
@@ -68,15 +72,38 @@ $ nix-build
 $ nix-shell --run 'cabal build'
```

Comment:
Could we also put `nix-build && ./result/bin/example` here too, just in case

Reply:
it is here, just two lines below